z.sh

```diff
@@ -29,8 +29,20 @@
 }
 
 _z() {
+    local DATA_FILE_NAME="z"
+    local DIRECTORY_NAME="z"
 
-    local datafile="${_Z_DATA:-$HOME/.z}"
+    # fallback to default XDG directory
+    local DEFAULT_DATA_DIR="$HOME/.local/share/$DIRECTORY_NAME"
```

Comment:
Maybe it is rather state than share regarding the latest XDG standard?

Reply:
I will change it - seems like a good case for state you are right.

Reply:
@Konfekt updated in my latest commit.